.travis.yml

```diff
@@ -260,6 +262,7 @@ jobs:
     - WEB3_TRANSACTION_CONFIRMATION_BLOCKS=1
     - AWS_ACCESS_KEY_ID=$ACCESS_KEY_PRD
     - AWS_SECRET_ACCESS_KEY=$SECRET_ACCESS_KEY_PRD
+    - NO_TEST_HOOKS=1
```

Comment:
Should we add `NO_TEST_HOOKS` to `webpack.config.js` as well?

Reply:
`EnvironmentPlugin`? No need to. Real environment variable is used to instruct babel to include the plugin or not. We don't need a const within the react app.